pkg/util/paramtable/autoindex_param.go

```diff
@@ -54,6 +55,12 @@ func (p *autoIndexConfig) init(base *BaseTable) {
 	}
 	p.IndexParams.Init(base.mgr)
 
+	p.PrepareParams = ParamItem{
```

Comment:
No default value ?

Reply:
Yes, default is an empty map, and knowhere will choose the default setting.